server/routes/handlers/helpers/dates.ts

```diff
@@ -137,15 +137,62 @@ export const formatDateTimeFromIsoString = (isoDate: string) => {
   }
   try {
     const includeTime = isoDate.length > 10
-    const date = DateTime.fromISO(isoDate, { zone: 'utc' }).setZone(timeZone)
+    const dateTime = getDateTimeInEuropeLondon(isoDate)
 
     if (includeTime) {
-      return date.toFormat("d MMMM yyyy' at 'HH:mm")
+      return dateTime.toFormat("d MMMM yyyy' at 'HH:mm")
     }
-    return date.toFormat('d MMMM yyyy')
+    return dateTime.toFormat('d MMMM yyyy')
   } catch (err) {
     return ''
   }
 }
 
 export const dateHasError = (field: string | DateValidationError) => Boolean((field as DateValidationError).error)
+
+export const recallAssessmentDueText = (isoDate?: string): string | undefined => {
+  if (!isDefined(isoDate)) {
+    return undefined
+  }
+  try {
+    return recallAssessmentDueString(getDateTimeUTC(isoDate))
+  } catch (err) {
+    logger.error(err)
+    return undefined
+  }
+}
+
+export const recallAssessmentDueString = (dueDateTime: DateTime): string => {
+  const now = DateTime.now()
+  const overdue = now.diff(dueDateTime).toMillis() >= 0
+  const startOfToday = now.set({ hour: 0, minute: 0, second: 0, millisecond: 0 })
+  const earlierToday = overdue && startOfToday.diff(dueDateTime).toMillis() < 0
+  const twentyFourHours = Duration.fromISO('PT24H')
+  const endOfToday = startOfToday.plus(twentyFourHours)
+  const afterTomorrow = endOfToday.plus(twentyFourHours).diff(dueDateTime).toMillis() < 0
+  const afterToday = afterTomorrow || endOfToday.diff(dueDateTime).toMillis() < 0
+  const dueForPresentation = dueDateTime.setZone(europeLondon)
+  const dueDate = dueForPresentation.toFormat('d MMMM yyyy')
```

Comment:
We're using this date format string in more than one place, could be worth centralising as a constant (later if you like)

Reply:
Done.